packages/components/pager/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - pager</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-pager--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.pager">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.pager" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.pager?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.pager" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.pager">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.pager" height="20"/>
+		</a>
+</p>
+
```

Comment:
Would be nice to add table with available properties.
Probably, it would be a good place to add some info about translations?